stumpy/gpu_stump.py

```diff
@@ -854,12 +764,12 @@ def gpu_stump(
         core._merge_topk_PI(profile[0], profile[i], indices[0], indices[i])
 
         # Update (top-1) left matrix profile and matrix profile indices
-        mask = profile_L[0] < profile_L[i]
+        mask = profile_L[0] > profile_L[i]
```

Comment:
This is another issue that I caught it myself. I think the test suite runs on 1 GPU (?) and that is why it couldn't  catch this mistake. I just noticed it and fixed it.